### ISSUE TITLE
"/home" command cooldown timer resets when logout/login

### ISSUE THREAD

**Issue by juniorjunior** [SimpleServer/SimpleServer#252]
If a user has a cooldown set for their access level and they use the "/home" command to port to their home the command works as expected. They get ported to their home and the cooldown timer starts. If they try to use the "/home" command again they get a message saying they have to wait the appropriate time. However, if they then log out and then back in they are able to use the /home command again immediately. It is as if the cooldown timer is being reset (an observation).

**Comment by dola** [comment]
This is true. Player objects are generated when a player joins a server and destroyed as soon as he leaves. Those include all information about a player that isn't permanent such as the cooldown time.

A simple possibility to come around this would be to start the cooldown every time a player joins. However this would affect all players and would make it impossible to instantly use warp each time you join until the cooldown time is over. The question is if this is tradeoff can be accepted.

I personally think it's better to call on fairness under the players. They should just all accept the regulations of warping around and if they don't you can always demote them to a lower group without the possibility to use the home command...

**Comment by NiLon** [comment]
Isn't simpleservers own storage suitable for storing cooldown timer?
